go/cmd/vtbench/vtbench.go

```diff
@@ -86,89 +83,97 @@ import (
 */
 
 var (
-	// connection flags
-	host       = flag.String("host", "", "vtgate host(s) in the form 'host1,host2,...'")
-	port       = flag.Int("port", 0, "vtgate port")
-	unixSocket = flag.String("unix_socket", "", "vtgate unix socket")
-	protocol   = flag.String("protocol", "mysql", "client protocol, either mysql (default), grpc-vtgate, or grpc-vttablet")
-	user       = flag.String("user", "", "username to connect using mysql (password comes from the db-credentials-file)")
-	db         = flag.String("db", "", "db name to use when connecting / running the queries (e.g. @replica, keyspace, keyspace/shard etc)")
-
-	// test flags
-	deadline = flag.Duration("deadline", 5*time.Minute, "maximum duration for the test run (default 5 minutes)")
-	sql      = flag.String("sql", "", "sql statement to execute")
-	threads  = flag.Int("threads", 2, "number of parallel threads to run")
-	count    = flag.Int("count", 1000, "number of queries per thread")
+	host, unixSocket, user, db, sql string
+	port                            int
+	protocol                        = "mysql"
+	deadline                        = 5 * time.Minute
+	threads                         = 2
+	count                           = 1000
 )
 
-func main() {
-	logger := logutil.NewConsoleLogger()
-	flag.CommandLine.SetOutput(logutil.NewLoggerWriter(logger))
+func initFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&host, "host", "", "vtgate host(s) in the form 'host1,host2,...'")
+	fs.IntVar(&port, "port", 0, "vtgate port")
```

Comment:
It is isn't strictly part of the flags move, but we want to start capitalizing the help messages

Reply:
I'll change that